pkg/inventory/model/client.go

```diff
@@ -52,7 +51,7 @@ type Client struct {
 	labeler Labeler
 	// The sqlite3 database will not support
 	// concurrent write operations.
-	dbMutex sync.Mutex
+	dbMutex sync.RWMutex
```

Comment:
This fixes the _"database locked"_ error that occurs when a goroutine performs a write operation (insert, update, delete, commit) while another is iterating a `Rows` (cursor).  The solution is to use a RW mutex.  The RLock() is acquired by the List() methods.